src/layers/VivViewerLayer/VivViewerLayer.js

```diff
@@ -77,6 +81,8 @@ export default class VivViewerLayer extends CompositeLayer {
       updateTriggers: {
         getTileData: [loader, loaderSelection]
       },
+      // A single tileSize x tileSize tile for Safari or null otherwise (the default)
+      maxCacheSize: NO_WEBGL2 ? 1 : null,
```

Comment:
Why do we need to set the cache size ?

Reply:
I removed it since the size difference between the two incoming data arrays is no longer so considerable.